src/hooks/bet/actions.ts

```diff
@@ -21,7 +21,7 @@ export const betTransaction = authenticatedAction
   .input(
     z.object({
       betAmount: z.number().min(1),
-      betResult: z.enum(["win", "loss"]),
+      betResult: z.enum(["win", "loss", "tie"]),
```

Comment:
เหมือนกัน

Reply:
แก้แล้ว
[fix: use betResult loss instead of tie](https://github.com/beambeambeam/modbid888/pull/36/commits/1a80558ad885a49f7426e29a1fc5133ea8b262db)